sampling:
  ros__parameters:
    leaf_size: 0.1 #다운샘플링 사이즈로 0.01은 1cm를 의미
roi_straight:
  ros__parameters:
    height_limit: 0.0 #높이제한
    left_limit: -3.0
    right_limit: 3.0
    front_limit: 0.0
    back_limit: -12.0 # back이 앞으로 되어 있음 
ransac: 
  ros__parameters:
    max_iterations: 20 #ransac알고리즘이 최적의 모델을 찾기 위해 시도하는 반복 횟수, 값이 클수록 더 정확한 모델을 찾을 수 있지만 계산 증가 시간이
    distance_threshold: 0.15 #데이터 포인트가 모델에 속하는 것으로 간주하는 최대거리, 값이 작으면 작을수록 더 정확한 모델을 얻을 수 있지만 노이즈에 더 민감해짐
dbscan:
  ros__parameters:
    cluster_tolerance: 0.3 #pcd 사이의 최대 거리
    min_cluster_size: 2 #pcd의 최소 개수
    max_cluster_size: 25 #pcd의 최대 개수
    min_x: 0.01 #최소 x좌표차
    max_x: 0.40 #최대 x좌표차
    min_y: 0.01
    max_y: 0.40
    min_z: 0.01
    max_z: 0.75
